src/main/java/controller/TimetableController.java

```diff
@@ -37,8 +39,8 @@ public void createTimetable (int busId, int locationId, String weekDay, String a
         Statement statement = databaseHandler.createStatement();
 
         try {
-            String newTimetable = "INSERT INTO timetable (locationId, weekDay, arrivalTime) VALUES ("
-                    + busId + "," + locationId + "," + "\"" + weekDay + "\"," + "\"" + arrivalTime + "\")";
+            String newTimetable = "INSERT INTO timetable (weekDay, arrivalTime, locationId, busId) VALUES ("
```

Comment:
I would rewrite it in this style
`String newTimetable = "INSERT INTO timetable (weekDay, arrivalTime, locationId, busId) VALUES" +
                    "('" + weekDay + "', '" + arrivalTime + "', '" + locationId + "', '" + busId + "')";`

Reply:
'-s gave different results then " for me when I was working in MYSQL Workbench, that's why I chose to go with "